tools/src/main/java/org/dashj/platform/tools/WalletTool.java

```diff
@@ -597,7 +624,19 @@ public static void main(String[] args) throws Exception {
             case ROTATE: rotate(); break;
             case SET_CREATION_TIME: setCreationTime(); break;
             // dashpay related commands
-            case CREATE_USERNAME: createUsername(waitForFlag); break;
+            case CREATE_USERNAME: {
+                boolean isCoinJoin = options.has("coinjoin");
+                if (!options.hasArgument("username")) {
+                    System.err.println("You must specify the username to create: --username=name");
+                    break;
+                }
+                Coin credits = Coin.valueOf(15000000);
+                if (options.hasArgument("amount")) {
+                    credits = parseCoin(options.valueOf(mixAmountFlag));
+                }
+                createUsername(waitForFlag, usernameFlag.value(options), credits, isCoinJoin);
+            } break;
```

Comment:
In DashPay the default value for the Upgrade fee (`credits`) is 0.15 DASH (0.15000000).  

Though we can also specify a lower amount, which was useful when testing coinjoin as part of the process.